### ISSUE TITLE
Steam controller does not work by cable In Windows Version

### ISSUE THREAD

**Issue by ManuelMT21An** [kozec/sc-controller#461]
[I] Daemon       Starting SC Controller Daemon v0.4.5...
[I] Daemon       Created control socket C:\Users\Manuel\AppData\Roaming\scc\daemon.socket
[I] Daemon       Initializing drivers...
[D] Mapper       Creating virtual devices...
[d] Mapper       Gamepad:  <ViGEm x360 Controller 0x02a6baa0>
[d] Mapper       Mouse:    <Winapi mouse device 0x012be880>
[d] Mapper       keyboard: <Winapi keyboard device 0x012be9b0>
[I] Daemon       Activated default profile 'C:\sc-controller-0.4.9.4-win32-pre-release\share\default_profiles\Desktop.sccprofile'
[E] USB          Failed to open device 1 on bus 6: No such device (it may have been disconnected)
[E] sc_by_cable  Failed to open '/win32/usb/6/1'
[I] Daemon       Ready.

**Comment by kozec** [comment]
Is that Windows 10?

**Comment by ManuelMT21An** [comment]
Yes

**Comment by kozec** [comment]
Sorry for responding so late, I was getting my hands on real hardware with W10. Thing is, I cannot replicate this at all.

Does it happen every time? Are you doing anything interesting, like having defective cable or Steam running in background?

**Comment by kozec** [comment]
I tagged this as "help wanted", because not only I was not able to reproduce the issue, even if I were, I'm not really sure how would I go around fixing it. It looks like something weird happening between libusb and Windows.

**Comment by kazoooka** [comment]
I've got the same problem 

[I] Daemon       Starting SC Controller Daemon v0.4.5...
[I] Daemon       Created control socket C:\Users\xxx\AppData\Roaming\scc\daemon.socket
[I] Daemon       Initializing drivers...
[D] Mapper       Creating virtual devices...
[d] Mapper       Gamepad:  <ViGEm x360 Controller 0x02a1baf0>
[d] Mapper       Mouse:    <Winapi mouse device 0x0090e9c0>
[d] Mapper       keyboard: <Winapi keyboard device 0x0090eaf0>
[I] Daemon       Activated default profile 'C:\Users\xxx\Downloads\sc-controller-0.4.9.4-win32-pre-release\share\default_profiles\Desktop.sccprofile'
[E] USB          Failed to open device 15 on bus 2: No such device (it may have been disconnected)
[E] sc_by_cable  Failed to open '/win32/usb/2/f'
[I] Daemon       Ready.

Let's connect somehow - maybe on a discord server and I try to give you as much information as possible!

**Comment by kozec** [comment]
I'm currently away from my PC, but quick question: Does anyone affected by issue connecting controller to  usb hub, or using multiple usb controllers (as in adding cards)? I'm noticing pattern of having bus number in `USB Failed to open device 15 on bus 2` anything but 1 here.

**Comment by rusagent** [comment]
Yes, i am using a USB Hub. And i kind of have to because i don't have much USB slots in my laptop :/

**Comment by rusagent** [comment]
There'I think there's a fix in an updated usblib https://github.com/libusb/libusb/issues/295

**Comment by notNSANE** [comment]
I'm not sure if this is the same issue (I'm new to the software) but I'm not running the SC on a USB Hub and all I can do is use the Lizard Mode on the SC. The green icon is enabled.

**Comment by kozec** [comment]
Just as report; I was able to replicate the issue, but I'm still not sure how to tackle it.

Turns out that wired SC is grabbed by "generic HID mouse / keyboard driver" in Windows, [which imposes some weird restrictions](https://github.com/libusb/libusb/blob/dea5a8e96891134829b97a75adfdf5586f908b6d/libusb/os/windows_winusb.c#L3575). Libusb then tries some magic and opens interface 0 (I have no idea why that one) and fails with FILE_NOT_FOUND. Nothing about this is right and I may end up patching libusb :(

So, this will be known bug for next pre-release.
